src/chord/main.cpp

```diff
@@ -68,4 +69,4 @@ int32 main(int32 argc, char ** argv)
 	} while(c != 'q');
 
 	return 0;
-}
\ No newline at end of file
+}
```

Comment:
you have not added anything in this file. So rebase this file.

Reply:
please remove unnecessary spacing in this file.